### PR TITLE
docs(README): use "Discord server" instead of "Server"

### PR DESCRIPTION
**Pull request by GodderE2D**
**Please describe the changes this PR makes and why it should be merged:**

I don't know about you, but **discord.js Server** doesn't sound right to me, and it doesn't describe what type of server it is. Using **discord.js Discord server** adds a description to it, and makes it clearer for the readers to know what a Discord server is used for discord.js help.

**Status and versioning classification:**

- Code changes have been tested against the Discord API, or there are no code changes
- I know how to update typings and have done so, or typings don't need updating
- This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

**Comment by iCrawl** [comment]
🤨

It's a discord api library, what other server are we expecting here? Gitter? Slack? Guilded? lol

**Comment by GodderE2D** [comment]
> 🤨
> 
> It's a discord api library, what other server are we expecting here? Gitter? Slack? Guilded? lol

Guilded.js soon? 👀

**Comment by Jxckson-ctrl** [comment]
TL;DR, this probably won’t get merged, and you should stop the pull request farming.


Picture this, if you call it a “Discord Server” wouldn’t it confuse a lot of people? As Crawl explained in the previous comment, it’s a Discord API library, there isn’t any other names for a server except Guild, in Discord.

Lastly, why in the world, would the discord.js developers want to make another package that would do the same things? I don’t think that would happen.

**Comment by GodderE2D** [comment]
> TL;DR, this probably won’t get merged, and you should stop the pull request farming.
> 
> Picture this, if you call it a “Discord Server” wouldn’t it confuse a lot of people? As Crawl explained in the previous comment, it’s a Discord API library, there isn’t any other names for a server except Guild, in Discord.
> 
> Lastly, why in the world, would the discord.js developers want to make another package that would do the same things? I don’t think that would happen.

1. Googling doesn't come up with any results for "pull request farming", so I'm going to assume that you're thinking I'm creating a mass amount of pull requests. And there's nothing wrong with that, many other people have opened a mass amount of pull requests. What matters is that you learn from your previous contributions and create better contributions in the future.
2. It's a nice addition to have, can't hurt to add another word to make it clearer, even if it only confuses 1 out of 1000 people.
3. Even if this shouldn't be changed, the letter s in "Server" shouldn't be capitalised as it isn't a proper noun.
3. It was a reference to [this](https://discord.com/channels/222078108977594368/222078108977594368/866370691863543828).
4. aren't*

**Comment by thehairy** [comment]
>     4. It was a reference to [this](https://discord.com/channels/222078108977594368/222078108977594368/866370691863543828).

https://discord.com/channels/222078108977594368/222078108977594368/869949365890981889 :3

**Comment by gamertike** [comment]
> > 🤨
> > It's a discord api library, what other server are we expecting here? Gitter? Slack? Guilded? lol
> 
> Guilded.js soon? 👀

There is [guilded.ts](https://github.com/guildedts/guilded.ts) 👀
